crates/sui-framework/packages/sui-framework/sources/priority_queue.move

```diff
@@ -69,7 +69,14 @@ module sui::priority_queue {
         };
         res
     }
-
+    //Return the length of queue
+    public fun length<T: drop>(pq: &PriorityQueue<T>): u64 {
+        pq.entries.length()
+    }
+    // Return `true` if the entries has no elements and `false` otherwise.
```

Comment:
Nit: can you please turn the comment into a doc one by starting it with `///` (3 fwd slashes)?
```suggestion
    /// Return `true` if the entries has no elements and `false` otherwise.
```

Reply:
Thank you for your suggestion. It has been modified